game/zz_hotkey_buttons.rpy

```diff
@@ -1,282 +1,295 @@
+    activate_sound gui.activate_sound
+
+style talk_button_text is default:
+    properties gui.button_text_properties("talk_button")
+    outlines []
+
+style hkbd_vbox is hkb_vbox
+style hkbd_button is button
+style hkbd_button_text is button_text
+
+style hkbd_vbox:
+    spacing 0
+
+style hkbd_button is default:
+    properties gui.button_properties("hkb_button")
+    insensitive_background "mod_assets/buttons/hkb_disabled_background.png"
+    idle_background "mod_assets/buttons/hkb_idle_background.png"
+    hover_background "mod_assets/buttons/hkb_hover_background.png"
+    hover_sound gui.hover_sound
+    activate_sound gui.activate_sound
+
+style hkbd_button_text is default:
+    font gui.default_font
+    size gui.button_text_size
+    xalign 0.5
+    idle_color "#e2d1d1"
+    hover_color "#FF8ED0"
+    outlines []
+
+style hkbd_button_black is default:
+    properties gui.button_properties("hkb_button_black")
+    idle_background "mod_assets/buttons/hkb_idle_background.png"
+    hover_background "mod_assets/buttons/hkb_idle_background.png"
+
+style hkbd_button_text_black is default:
+    font gui.default_font
+    size gui.button_text_size
+    xalign 0.5
+    idle_color "#e2d1d1"
+    hover_color "#FF8ED0"
+    outlines []
+
+style hkbd_label is default:
+    properties gui.button_properties("hkb_label")
+    background "mod_assets/buttons/choice_hover_blank.png"
+
+style hkbd_label_text is default:
+    font gui.default_font
+    size 20
+    xalign 0.5
+    color "#e2d1d1"
+    outlines []
+
+style hkbd_option_text is default:
+    font gui.default_font
+    size gui.button_text_size
+    xalign 0.5
+    idle_color "#e2d1d1"
+    hover_color "#FF8ED0"
+    outlines []
+
+style hkbd_option is default:
+    properties gui.button_properties("hkb_option")
+    idle_background "mod_assets/buttons/option_hover_blank.png"
+    hover_background "mod_assets/buttons/option_hover_fold.png"
+    hover_sound gui.hover_sound
+    activate_sound gui.activate_sound
+
+style talkd_vbox is vbox
+style talkd_button is button
+style talkd_button_text is button_text
+
+style talkd_vbox_black is vbox
+style talkd_button_black is button
+style talkd_button_text_black is button_text
+
+style talkd_vbox:
+    spacing 0
+
+style talkd_button is default:
+    properties gui.button_properties("hkb_button")
+    idle_background "mod_assets/buttons/hkb_disabled_background.png"
+    hover_background "mod_assets/buttons/hkb_disabled_background.png"
+
+style talkd_button_text is default:
+    font gui.default_font
+    size gui.button_text_size
+    xalign 0.5
+    idle_color "#e2d1d1"
+    hover_color "#FF8ED0"
+    outlines []
+
+screen hkb_overlay:
+    zorder 50
+
+    style_prefix "hkb"
+
+    key "t" action [ Jump("talk_menu"),
+                SensitiveIf(not jn_globals.player_is_in_conversation and not jn_globals.player_is_ingame) 
+            ]
+
+    if persistent.jn_custom_music_explanation_given and Natsuki.isHappy(higher=True):

```

Comment:
Perhaps we could move these conditions into the `SensitiveIf` check? It would help keep things cleaner by having everything in a single check!